nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/pages/flow-designer/service/manager/processor-manager.service.ts

```diff
@@ -669,20 +669,20 @@ export class ProcessorManager {
         // update the run status
         updated
             .select('text.run-status-icon')
-            .attr('fill', function (d: any) {
-                let fill: string = '#728e9b';
+            .attr('class', function (d: any) {
+                let clazz: string = 'primary-500';
 
                 if (d.status.aggregateSnapshot.runStatus === 'Validating') {
-                    fill = '#a8a8a8';
+                    clazz = 'warn-contrast-300';
                 } else if (d.status.aggregateSnapshot.runStatus === 'Invalid') {
-                    fill = '#cf9f5d';
+                    clazz = 'accent-A200';
                 } else if (d.status.aggregateSnapshot.runStatus === 'Running') {
-                    fill = '#7dc7a0';
+                    clazz = 'accent-200';
                 } else if (d.status.aggregateSnapshot.runStatus === 'Stopped') {
-                    fill = '#d18686';
+                    clazz = 'warn-200';
                 }
 
-                return fill;
+                return clazz;
```

Comment:
I think this is overwriting the `run-status-icon` class already on this element.
It seems to be preventing the icon from updating when the processor is started/stopped.

Ports have the same issue.

probably want something like:
```
            .attr('class', function (d: any) {
                let clazz: string = 'primary-500';

                if (d.status.aggregateSnapshot.runStatus === 'Validating') {
                    clazz = 'warn-contrast-300';
                } else if (d.status.aggregateSnapshot.runStatus === 'Invalid') {
                    clazz = 'accent-A200';
                } else if (d.status.aggregateSnapshot.runStatus === 'Running') {
                    clazz = 'accent-200';
                } else if (d.status.aggregateSnapshot.runStatus === 'Stopped') {
                    clazz = 'warn-200';
                }

                return `run-status-icon ${clazz}`;
            })
```

Reply:
Great catch! I have updated these in my latest commit. Thanks!